python/tvm/meta_schedule/runner/config.py

```diff
@@ -0,0 +1,189 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Configurations for measurements in the runner"""
+import os
+from threading import Thread
+from typing import NamedTuple, Optional, Union
+
+from tvm import rpc
+
+
+class EvaluatorConfig(NamedTuple):
+    """Config Details of Evaluator
+
+    Parameters
+    ----------
+    number: int
+        The number of runs.
+    repeat: int
+        The number of times to repeat in each run.
+    min_repeat_ms: int
+        minimum repeat time in ms.
```

Comment:
I always feel this is a confuse term and I believe no one knows what does that mean by its name.
Maybe it's a good opportunity to improve it in this PR by changing it to something like

```suggestion
    adaptive_number_to: int
        if the execution latency is too short, increase the number of runs to the given time (in ms)
        to reduce the measurement error.
```

Reply:
I agree with the idea to improve the overall doc and naming. In terms of doc, I completely agree that the suggested change can greatly improve the users' understanding.

The naming choice `min_repeat_ms` is a separate issue - we dislike this name too, but we do want to be consistent with TVM's [time_evaluator](https://tvm.apache.org/docs/api/python/runtime.html#tvm.runtime.Module.time_evaluator) API. Therefore, I would say it's probably good for now to stick with the name `min_repeat_ms` here, and break it together with `time_evaluator` API change in the future